packages/internal/src/typeCheck.js

```diff
@@ -0,0 +1,23 @@
+// @ts-check
+import { mustMatch as typelessMustMatch } from '@endo/patterns';
```

Comment:
Should we eventually move this into @endo/patterns so the only `mustMatch` is the typed one, and the typeless one is never exported? (Modulo the transition costs, which I expect will be painful but tolerable.)

Reply:
Yes, I think once we've had some stress testing of it so we know it won't change much.